docs/examples.md

```diff
@@ -0,0 +1,33 @@
+# Examples
+Here you can find some examples of what some users have created in their personal keyboard configs. These are here to
+help you understand how some of the tools may be used.
+
+## Changing LED color based on layers
+This allows you to create a layer key that also changes colors when pushing a layer key, and restore turn off the lights
+when you release the layer key. The example uses the MO though any layer switch keys can be used if imported. Just use the
+LAYER_1 key in your keymap, and it's ready to go! You can change animations, colors, or anything in there.
+
+```python
+from kmk.handlers.layers import (mo_pressed, mo_released)
```

Comment:
this can now be updated to use `before_press_handler`/`before_release_handler` (or the `after` functions, whatever) and probably _should_ - the goal is for users to not need to import+call built-in handlers like this anymore

Reply:
Is this in the docs anywhere? That's not a thing that I'm familiar with unfortunately.